### ISSUE TITLE
Requirements for metadata validator

### ISSUE THREAD

**Issue by ackermag** [biocore/qiita#559]
Suggested list of functions for metadata validator:
Metadata validator:
- [ ] confirms that terms that should confirm to ontology;
- [ ] confirms that numerical samples are numerical;
- [ ] confirms that restricted list fields conform;
- [ ] confirms that host_subject_id is present for all host-associated studies;
- [ ] confirms that all required fields are present (suggestion);
- [ ] confirms that anything entered complies with EBI requirements (suggestion);

**Comment by ackermag** [comment]
Host_subject_id needs to be in all studies (not just host-associated) for EBI submission

**Comment by ackermag** [comment]
How will we deal with updates to ontologies?

**Comment by josenavas** [comment]
@ackermag why is host_subject_id required for EBI? The current documentation points out that is required for our centralized Qiita, but nor for EBI. Did any EBI submission failed due to missing host_subject_id? Last time that I checked host_subject_id was not used in the EBI submission code.

**Comment by ackermag** [comment]
Yes - previously submissions failed due to missing host_subject_id in
studies that were not host-associated.  I will test again on next
submission of non-host-associated study.


On Fri, Aug 7, 2015 at 9:44 AM, josenavas <notifications@github.com> wrote:

> @ackermag <https://github.com/ackermag> why is host_subject_id required
> for EBI? The current documentation points out that is required for our
> centralized Qiita, but nor for EBI. Did any EBI submission failed due to
> missing host_subject_id? Last time that I checked host_subject_id was not
> used in the EBI submission code.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/qiita/issues/559#issuecomment-128760289>.
>



-- 
Gail Ackermann
Knight Lab
UCSD
glackermann@ucsd.edu <ackermag@ucsd.edu>
